src/styles/block/catalog--card-stule.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  box-sizing: border-box;
+  height: 408px;
```

Comment:
do not hardcode height - it should depend on content and margins/paddings inside the element